src/components/procedure/_remove-ruleset.mdx

```diff
@@ -1,15 +1,22 @@
-To **disable a ruleset** from the Policies page:
+You can disable a specific rule or ruleset to prevent Semgrep Code from using it when scanning your codebase.
 
-1. In Semgrep Cloud Platform, click **Rules > [Policies](https://semgrep.dev/orgs/-/policies)**.
+:::info
+- When you disable a rule, all previous findings generated by that rule are still retained.
```

Comment:
Wait, so if a rule is **disabled**, findings are still **retained** - doesn't that contradict the **removed** state? I think there are 2 different states for rules, **disabled** and **removed** from the policy?

Reply:
Yes, but not that specific finding, so I think something like this makes more sense? "When you disable a rule, all *other* previous findings generated by that rule are still retained." (If this isn't supposed to be the case, then the UI is off)

Reply:
Can you check with the findings team? (You can @ Clara directly.) AFAIK we have a disable and remove state and I'm not sure if the user can do the "removed" state or if that's our internal detail when we update old rules that were replaced with new ones.

Reply:
Verified the change with Clara and confirmed via testing